OmsAgent/omsagent.py

```diff
@@ -124,6 +126,46 @@
 except:
     pass
 
+def verifyShellBundleSigningAndChecksum():
+    keys_directory = os.path.join(os.getcwd(), keysDirectory)
+    # import GPG key
+    dscGPGKeyFilePath = join(keys_directory, 'dscgpgkey.asc')
+    if not os.path.isfile(dscGPGKeyFilePath):
+        raise Exception("Unable to find the dscgpgkey.asc file at " + dscGPGKeyFilePath)
+    
+    importGPGKeyCommand = "sh ImportGPGkey.sh " + dscGPGKeyFilePath
+    importGPGKeyResult = subprocess.call(importGPGKeyCommand, shell = True)
```

Comment:
can we use run_command_with_retries_output() which is an existing utility?

Reply:
done